bank/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "bank"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+openssl = "0.10.41"
+postgres-openssl = "0.5.0"
+chrono = "0.4.22"
+
+[dependencies.postgres]
+version = "0.19.3"
+features = [
+    "with-uuid-1"
+]
+
+[dependencies.uuid]
+version = "1.1.2"
+features = [
+    "v4",                # Lets you generate random UUIDs
+    "fast-rng",          # Use a faster (but still sufficiently random) RNG
+    "macro-diagnostics", # Enable better diagnostics for compile-time UUIDs
+]
+
+[net]
+git-fetch-with-cli = true
```

Comment:
This is a pretty uncommon option, I would suggest removing it or commenting why it's needed.

Reply:
Done.